File: sum/sum.cpp
#include <iostream>

using namespace std;

int sum(int x, int y){
    return x + y;
}

int main(){
    int x, y;
    cin >> x >> y;
    cout << x << " + " << y << " = " << sum(x, y) << endl;
    return 0;
}

File: sum/sum.php
<?php 

/**
 * Calculate sum of numbers.
 * 
 * @param (string) $nums Numbers seperated with comma.
 * 
 * @return int
 */
function sum($nums)
{

    $answer = 0;
    $n = explode(",", $nums);
    foreach ($n as $key => $value) {
        $num = (int) $value;
        if (is_numeric($value)) {
            $answer += $value;
        }
    }

    return $answer;
}

var_dump(sum("25,25"));
